chain/jsonrpc/src/lib.rs

```diff
@@ -958,6 +968,26 @@ impl JsonRpcHandler {
     }
 }
 
+#[cfg(feature = "sandbox")]
+impl JsonRpcHandler {
+    async fn sandbox_patch_state(
+        &self,
+        patch_state_request: near_jsonrpc_primitives::types::sandbox::RpcSandboxPatchStateRequest,
+    ) -> Result<Value, RpcError> {
+        actix::spawn(
+            self.client_addr
+                .send(NetworkClientMessages::Sandbox(NetworkSandboxMessage::SandboxPatchState(
+                    patch_state_request.records,
+                )))
+                .map(|_| ()),
+        );
+        Ok(serde_json::to_value(
+            near_jsonrpc_primitives::types::sandbox::RpcSandboxPatchStateResponse {},
+        )
+        .unwrap())
```

Comment:
Please, see `health` endpoint as an example. Return the instance of the type itself:

```suggestion
    Ok(near_jsonrpc_primitives::types::sandbox::RpcSandboxPatchStateResponse {})
```

And in the `fn process_request` encode the result with `serde_json::to_value`, e.g., see `health` method:

https://github.com/near/nearcore/blob/322f6d80b9a6d0f15c3e374a5d368108166a2503/chain/jsonrpc/src/lib.rs#L278-L279